synapse/storage/databases/main/__init__.py

```diff
@@ -264,7 +264,7 @@ def _get_active_presence(self, db_conn):
 
         return [UserPresenceState(**row) for row in rows]
 
-    async def get_users(self) -> List[Dict[str, Any]]:
+    async def get_users(self) -> JsonDict:
```

Comment:
I think this still returns a list? So it should be `List[JsonDict]`? Same for the other changes in this file.

Reply:
I have made an update.